testing/python/metafunc.py

```diff
@@ -1780,3 +1780,37 @@ def test_foo(a):
         )
         result = testdir.runpytest()
         result.assert_outcomes(passed=1)
+
+    def test_parametrize_explicit_parameters_xfail(self, testdir):
+        testdir.makepyfile(
+            """
+            import pytest
+
+            @pytest.fixture
+            def fixture(arg):
+                return arg
+
+            @pytest.mark.parametrize("arg", ["baz"])
+            def test_without_argument(fixture):
+                assert "baz" == fixture
+        """
+        )
+        result = testdir.runpytest()
+        result.assert_outcomes(failed=1)
+
+    def test_parametrize_explicit_parameters(self, testdir):
+        testdir.makepyfile(
+            """
+            import pytest
+
+            @pytest.fixture
+            def fixture(arg):
+                return arg
+
+            @pytest.mark.parametrize("arg", ["baz"])
+            def test_with_argument(arg, fixture):
+                assert "baz" == fixture
+        """
+        )
+        result = testdir.runpytest()
+        result.assert_outcome(failed=1)
```

Comment:
This is not right, this test should pass. In fact we don't need this `test_parametrize_explicit_parameters` test at all, as this is the normal behavior which is already tested through the rest of the test suite.

Reply:
Agree! Fixed in new commit